readme.md

```diff
@@ -116,7 +116,8 @@ These links are generated relative to a project.
 In Node this is detected automatically by loading `package.json` and looking for
 a `repository` field.
 In the browser, or when overwriting this, you can pass a `repository` in
-`options`.
+`options`.  
+Note: `repository` expects a `user/repo` pairing.
```

Comment:
```suggestion
The value of `repository` should be a URL to a GitHub repository, such as
`'https://github.com/user/project.git'`, or only `'user/project'`.
```

This suggestion expands on the expected type: whole URLs are also supported!

Reply:
Ah neat!